src/utils/AuthProvider.tsx

```diff
@@ -97,13 +97,33 @@ export default function AuthProvider({
   // sign up and set the session, userId, and userEmail
   const signUp = useCallback(
     async (email: string, password: string, options: SignUpOptions) => {
-      const value = await supabase.auth.signUp({
+      const { data, error } = await supabase.auth.signUp({
```

Comment:
could we change this back to `const value = ...`? This will make it easier to return an `AuthResponse` as we were doing previously.